setup.py

```diff
@@ -58,8 +60,9 @@
     classifiers=[
         "License :: OSI Approved :: Apache Software License",
         "Programming Language :: Python :: 3 :: Only",
-        "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
```

Comment:
I just tested and some of the binary dependencies we are using haven't been built for 3.11 yet and error on `pip install`. Based on some of my initial testing with 3.11 I'm confident all of the fidesctl code itself will work, we just need to wait for binaries to be available to say we support it.

```suggestion
````